core/storage/auth/gae_models.py

```diff
@@ -199,6 +199,19 @@ def has_reference_to_user_id(cls, user_id):
         """
         return cls.query(cls.user_id == user_id).get(keys_only=True) is not None
 
+    @classmethod
+    def get_by_gae_id(cls, gae_id):
+        """Fetch an entry by user ID.
```

Comment:
Prefer: ``by GAE user ID`` to avoid potentially introducing confusion when compared with a standard Oppia user ID.